jbot/src/main/java/me/ramswaroop/jbot/core/facebook/Bot.java

```diff
@@ -221,6 +221,22 @@ public final void subscribeAppToPage() {
         restTemplate.postForEntity(fbApiEndpoints.getSubscribeUrl(), params, String.class);
     }
 
+    /**
+     * Call this method to send a message without a previous interaction of the user.
+     *
+     * @param userId id of user you want to send the message to
+     * @param messageText message that you want to send to user
+     */
+    protected final void sendMessageToUser(String userId, String messageText) {
```

Comment:
This method should be present in `FbService.java` instead. If you can move this method, this PR will be ready to be merged.